### DISCUSSION TITLE
spaCy 3.3 release on conda-forge?

### DISCUSSION THREAD

**Discussion by staedi** [explosion/spaCy#10737] [Help: Installation] [answered]
Hello,
First, I want to say thank you for the hard work on spaCy.

I was excited for the 3.3 release of spaCy given that it would be equipped with Korean Language model.
Unfortunately, the version on conda-forge doesn't match.

While the newest release date is the same as that of 3.3 (2 days ago), the version on the [anaconda.org](https://anaconda.org/conda-forge/spacy) shows the version as 3.2.3.

Since I've using mamba/conda primarily, I prefer to install from it.

Is that release on anaconda a mistake? If so, can you re-upload for the 3.3?

Thanks again!

**Comment by adrianeboyd** [comment, marked as answer]
There's always a bit of a delay because the conda-forge packages are built from the pypi release. In this case, we ran into additional errors building the windows packages, so it might take a bit longer. Hopefully not more than a few days, but I'm not sure at this point.
